### PR TITLE
Add Oxygen XML Editor.app v18.1

### PR DESCRIPTION
**Pull request by joewiz**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference].
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/oxygen-xml-editor.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'oxygen-xml-editor' do
+  version '18.1'
```
Comment (vitorgalvao) [inline Casks/oxygen-xml-editor.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/f25b6babcd398abf48e33af3d887b2d00de1d661/Casks/kibako.rb).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/oxygen-xml-editor.rb (hunk 1 above)
Comment (joewiz) [inline Casks/oxygen-xml-editor.rb]: @vitorgalvao Thank you, I've made the change. Still learning!

**Review by vitorgalvao — APPROVED** [review]

File: Casks/oxygen-xml-editor.rb (hunk 1 above)
Comment (joewiz) [inline Casks/oxygen-xml-editor.rb]: @vitorgalvao Sorry about the bad version clause, I'd made the change but neglected to add it before my 2nd commit. Thank you for catching it. I noticed the kibako example included a license clause, so I've added one for this software.

Comment (vitorgalvao) [inline Casks/oxygen-xml-editor.rb]: > I noticed the kibako example included a license clause

Where’s the example, so we can fix it?

Comment (vitorgalvao) [inline Casks/oxygen-xml-editor.rb]: Ah, that’s because I have that message in an automated snippet. I’ll update that.

**Comment by joewiz** [comment]
@miccal @vitorgalvao Thanks so much, as always, for your guidance and support!

**Comment by miccal** [comment]
Your welcome @joewiz, but @vitorgalvao did most of the work here, I just merged it.

**Pull request merged**